src/main/java/org/osm2world/console/ImageExporter.java

```diff
@@ -101,14 +94,11 @@ private ImageExporter(Configuration config, Results results,
 		}
 
 		if (config.containsKey(BG_IMAGE_KEY)) {
-			String fileString = config.getString(BG_IMAGE_KEY);
-			if (fileString != null) {
-				backgroundImage = new File(fileString);
-				if (!backgroundImage.exists()) {
-					System.err.println("background image file doesn't exist: "
-							+ backgroundImage);
-					backgroundImage = null;
-				}
+			backgroundImage = ConfigUtil.resolveFileConfigProperty(config, BG_IMAGE_KEY);
```

Comment:
Here you're passing in a property key instead of the value, probably a bug?

Reply:
Yep, that's a bug